unison-src/builtin-tests/interpreter-tests.tpl.md

```diff
@@ -1,5 +1,11 @@
+The runtime tests are hosted at https://share.unison-lang.org/@unison/runtime-tests/
+
+If you want to add or update tests, you can create a branch of that project, and update the `runtime_tests_version` line in `jit-tests.sh` and `CI.yaml`
+
+Before merging the PR on Github, we'll merge your branch on Share and restore `runtime_tests_version` to /main or maybe a release.
+
 ```ucm:hide:error
-.> delete.project nothing
+.> delete.project dont-delete
```

Comment:
Sorry, why are we deleting "dont-delete"? Hahaha

Reply:
@ChrisPenner Lol yeah, I guess that looks extra goofy with my last change.
It's because `delete.project runtime-tests` may trigger an error if it's the first run and it doesn't exist (the codebase is reused between runs to save time on pulling). We don't have any way to handle errors in transcripts, so I trigger a deterministic error followed by a sometimes error, and mark the whole block as error.
Maybe there's a better way to do it?  We don't have a `clone.force`, do we?

I could call the project `error-for-the-error-block`?

Reply:
You can do whatever you think is best, just wanted to point out that it looks a bit silly and I'd be confused if I were reading through it and saw that.

Reply:
For sure, ok thanks.

Reply:
For sure, ok thanks.